content/en/docs/architecture/step-registry.md

```diff
@@ -959,3 +959,59 @@ tests:
   steps:
     workflow: openshift-e2e-disruptive
 {{< / highlight >}}
+
+## VPN connection
+
+For platforms that need access to restricted environments, `ci-operator`
+supports adding a dedicated VPN connection to each test step.  Since this is a
+requirement of specific platforms, it is triggered by special files in the
+[cluster profile(s)]({{< ref "docs/how-tos/adding-a-cluster-profile#what-is-a-cluster-profile" >}})
+associated with those platforms.  This process is transparent to the test
+command: when a VPN connection is requested at the test level, it is set up
+automatically by the test platform with no changes required to individual tests.
+
+{{< alert title="Note" color="info" >}}
+Details of the implementation can be found in the
+<a href="https://docs.google.com/document/d/1mPjrHVS1EvmLdq4kGhRazTpGu6xVZDyGpVAphVZhX4w/edit?resourcekey=0-KA-qXXq1J2bTR7o6Kit9Vw">design document</a>.
+{{< /alert >}}
+
+### Cluster profile
+
+VPN connections are requested by the presence of special files in the cluster
+profile, detected when the test steps are about to be executed:
+
+- `vpn_image.txt`: _pull spec_ of the image to be used for the VPN client
+  container.  It should contain the files necessary to establish the connection
+  as well as the `bash` shell.
```

Comment:
"It should contain the files necessary to establish the connection
  as well as the `bash` shell."

Should it be with the 2nd bullet (about `vpn.sh`)?

Reply:
Oh. I see now. I misunderstood what "it" means.
The "it" is the container which includes files for VPN and `bash` environment.
It makes good sense to me now.

Reply:
This is now a field in `vpn.yaml`, I've also clarified the description.